lib/mayaUsdAPI/render.cpp

```diff
@@ -0,0 +1,160 @@
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+#include "render.h"
+
+#include <mayaUsd/render/MaterialXGenOgsXml/OgsFragment.h>
+#include <mayaUsd/render/MaterialXGenOgsXml/OgsXmlGenerator.h>
+#include <mayaUsd/render/MaterialXGenOgsXml/ShaderGenUtil.h>
+#include <mayaUsd/render/vp2RenderDelegate/colorManagementPreferences.h>
+
+namespace MAYAUSDAPI_NS_DEF {
+
+namespace ColorManagementPreferences {
+
+bool IsActive() { return MayaUsd::ColorManagementPreferences::Active(); }
+
+std::string getFileRule(const std::string& path)
+{
+    return MayaUsd::ColorManagementPreferences::getFileRule(path);
+}
+
+} // namespace ColorManagementPreferences
+
+namespace OgsXmlGenerator {
+
+std::string textureToSamplerName(const std::string& textureName)
+{
+    return MaterialX::OgsXmlGenerator::textureToSamplerName(textureName);
+}
+
+void setUseLightAPI(int val) { MaterialX::OgsXmlGenerator::setUseLightAPI(val); }
+
+const std::string& getPrimaryUVSetName()
+{
+    return MaterialX::OgsXmlGenerator::getPrimaryUVSetName();
+}
+
+void setPrimaryUVSetName(const std::string& val)
+{
+    MaterialX::OgsXmlGenerator::setPrimaryUVSetName(val);
+}
+
+} // namespace OgsXmlGenerator
+
+namespace OgsFragment {
+
+std::string getMatrix4Name(const std::string& matrix3Name)
+{
+    return MaterialXMaya::OgsFragment::getMatrix4Name(matrix3Name);
+}
+
+std::string registerOCIOFragment(const std::string& fragName)
+{
+    return MaterialXMaya::OgsFragment::registerOCIOFragment(fragName);
+}
+
+MaterialX::DocumentPtr getOCIOLibrary() { return MaterialXMaya::OgsFragment::getOCIOLibrary(); }
+
+struct OgsFragmentImpl
+{
+    OgsFragmentImpl(MaterialX::ElementPtr ptr, const MaterialX::FileSearchPath& librarySearchPath)
+        : _ogsFragment(ptr, librarySearchPath)
+    {
+    }
+
+    const MaterialXMaya::OgsFragment _ogsFragment;
+};
+
+OgsFragment::OgsFragment(
+    MaterialX::ElementPtr            ptr,
+    const MaterialX::FileSearchPath& librarySearchPath)
+    : _imp(new OgsFragmentImpl(ptr, librarySearchPath))
+{
+}
+
+// When using a pimpl you need to define the destructor here in the
+// .cpp so the compiler has access to the impl.
+OgsFragment::~OgsFragment() = default;
+
+const std::string& OgsFragment::getFragmentSource() const
+{
+    return _imp->_ogsFragment.getFragmentSource();
+}
+
+const std::string& OgsFragment::getFragmentName() const
+{
+    return _imp->_ogsFragment.getFragmentName();
+}
+
+const MaterialX::StringMap& OgsFragment::getPathInputMap() const
+{
+    return _imp->_ogsFragment.getPathInputMap();
+}
+
+std::string OgsFragment::getMatrix4Name(const std::string& matrix3Name)
+{
+    return _imp->_ogsFragment.getMatrix4Name(matrix3Name);
+}
+
```

Comment:
I added a new function very recently, you might want to also export `MaterialXMaya::OgsFragment::isTransparentSurface()`

Reply:
Thanks. I've synced with latest LookdevX and added the missing function to this PR.